bindings/python/triplestore/backends/rdflib.py

```diff
@@ -92,8 +92,17 @@ def serialize(self, destination=None, format='turtle', **kwargs):
             return s if isinstance(s, str) else s.decode()
 
     def query(self, query_object, **kwargs):
-        """SPARQL query."""
-        return self.graph.query(query_object=query_object, **kwargs)
+        """SPARQL query.
+
+        Parameters:
+            query_object: String with the SPARQL query.
+            kwargs: Keyword arguments passed to rdflib.Graph.query().
+
+        Returns:
+            List of tuples of IRIs for each matching row.
+        """
+        rows = self.graph.query(query_object=query_object, **kwargs)
+        return [tuple(str(v) for v in row) for row in rows]
```

Comment:
Should the query API be restricted to SELECT statements? If not we might have to consider returning a Results-object?

Reply:
Good point. INSERT and DELETE queries are expected to be used with the update() method since they modifies the triplestore. We should document that. As I understand is this how rdflib does it.

What other types of queries does there exists?

Reply:
I have updated the docstrings